app/controllers/admin/partners_controller.rb

```diff
@@ -52,8 +58,16 @@ def partner
       existing_publisher || Partner.new(email: email_params)
     end
 
+    def organization
+      Organization.find_by(name: params[:organization_name]) || Organization.new(name: params[:organization_name])
```

Comment:
Let's do this instead
```suggestion
      Organization.find_or_initialize_by(name: params[:organization_name])
```